application/dash_application/views/__init__.py

```diff
@@ -6,11 +6,12 @@ def navbar(current):
         children=[
             dbc.DropdownMenu(
                 children=[
-                    dbc.DropdownMenuItem("Exome", href="exome", style={"fontSize": "12pt"}),
-                    dbc.DropdownMenuItem("Pre-Exome", href="preexome", style={"fontSize": "12pt"}),
-                    dbc.DropdownMenuItem("sWGS", href="swgs", style={"fontSize": "12pt"}),
-                    dbc.DropdownMenuItem("Pre-WGS", href="prewgs", style={"fontSize": "12pt"}),
-                    dbc.DropdownMenuItem("CFMEDIP", href="cfmedip", style={"fontSize": "12pt"}),
+                    dbc.DropdownMenuItem("Pre-Exome", href="preqc-exome",
```

Comment:
for future consideration - I think we could generate this list by iterating over the page_names from pages.py so we don't have to update this manually

Reply:
It would be nice to do `index.html` the same way.

Reply:
`index.html` is I think meant to be rewritten with some nice looking icons eventually...